app/views/users/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>The Users</h1>
```

Comment:
- [ ] Just as you have the users index page, you should also have the user show page. Kindly create the file.